README.md

```diff
@@ -53,7 +53,8 @@ can be connected to a Fire TV at a time. Note that while you can install on an
 Android TV emulator, the behavior is different from Fire TV's and should not be
 relied upon.
 
-When using an emulator, you can press `cmd+m` to simulate a menu button press.
+If using an emulator, we recommend the Android TV device image: either 720p or
```

Comment:
Nit: Also include the API level - for now, the Fire TV stick/diamond run r22 and r25 respectively.